classes/class-wc-rest-connect-self-help-controller.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+if ( ! defined( 'WOOCOMMERCE_CONNECT_MAX_SELF_HELP_DEPTH' ) ) {
+	define( 'WOOCOMMERCE_CONNECT_MAX_SELF_HELP_DEPTH', 32 );
```

Comment:
Fixed in https://github.com/Automattic/woocommerce-connect-client/pull/376/commits/77e6c33e877d6931f17e4958e855171bc1cf814b